packages/sandboxd/spec/filesystem/filesystem.proto

```
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package sandboxd.filesystem.v1;
```

Buf (1.70.0)

[error] Files with package "sandboxd.filesystem.v1" must be within a directory "sandboxd/filesystem/v1" relative to root but were in directory "packages/sandboxd/spec/filesystem".

(PACKAGE_DIRECTORY_MATCH)